test_requirements.txt

```diff
@@ -4,6 +4,7 @@ h5py>=2.10.0
 requests>=2.23.0
 statsmodels>=0.11.0
 cma>=2.7.0
+pydub>=0.23,<0.24
```

Comment:
Should we add pydub to requirements.txt?

Reply:
The `pydub` dependency is only used when someone wants to apply the `Mp3Compression` defense. It is my understanding that such less-used dependencies are the users responsibility. E.g. like the `PIL` dependency in the `JpegCompression`.